Dockerfile

```diff
@@ -1,49 +1,43 @@
 #default image to ubuntu + install rocm
-ARG BASEIMAGE=ubuntu:20.04
-ARG ROCM_PRE=0
-#ARG IMG_VER=$([[ $BASEIMAGE == "ubuntu:20.04" ]]; echo $?)
+ARG BASEIMAGE=rocm/miopen:ci_5450cc
 
-FROM ubuntu:20.04 as dtuna-ver-0
+#FROM ubuntu:20.04 as dtuna-ver-0
+FROM $BASEIMAGE as dtuna-ver-0
 #install rocm
 ARG ROCMVERSION=
-ARG OSDB_BKC_VERSION='12969'
+ARG OSDB_BKC_VERSION=
+#'12969'
+ENV NO_ROCM_INST=
 # Add rocm repository
-RUN apt-get update
-RUN apt-get install -y wget gnupg
+RUN apt-get update && apt-get install -y wget gnupg
 RUN wget -qO - http://repo.radeon.com/rocm/rocm.gpg.key | apt-key add -
-RUN if ! [ -z $OSDB_BKC_VERSION ]; then \
+RUN echo "" > /env; \
```

Comment:
I'm still new to bash scripting. I wonder why `echo "" > /env;` is needed?

Reply:
Each of the RUN statements is executed in a separate environment. To carry over env vars I'm passing around this file that can be executed at the start of a statement. However the file should exist or there will be a failure.

Reply:
Quick question: can we do `export ENV_VAR=123` for it to be carried over to all the child processes? We can do it in a separate PR if that's something that can help -- maybe prevent the build from crashing if the env file doesn't exist?